code-injection-analysis/Papa-Pedro-13_react-store/node.js

```
@@ -0,0 +1,47 @@
+
+  const url3 = req.query.url.includes("http") ? req.query.url : '';
+  let response2 = await fetch(url3);
+
+  res.send(response.data);
+});
+
+
+//Eval
+app.get('/execute', (req, res) => {
+  const code = req.query.script;
+  eval('console.log(123)'); // Опасный вызов
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)